Orchard1.ruleset

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Orchard 1 C# rules" ToolsVersion="16.0">

```

Comment:
This can be probably disabled, since Orchard validates all requests by default? https://rules.sonarsource.com/csharp/RSPEC-4564 As well as SCS0016. "SCS0017 C# Request validation is disabled" can remain as it's good to double-check.

Reply:
I was impatient and implemented this :).